.github/ci_templates/helper_jobs.yml

```diff
@@ -39,6 +40,8 @@ git_info:
       uses: {!{ index (ds "actions") "actions/github-script" }!}
       with:
         script: |
+          core.debug(`Event: ${context.eventName}`)
+          core.debug(`Inputs: ${JSON.stringify(context.payload.inputs)}`)
```

Comment:
No need after developing, there is a "debug mode" switch when job is restaretd and "always debug mode" in test repositories.